docs/massivekb.md

```diff
@@ -19,6 +42,66 @@ These spectral libraries can be:
+
+1. Database Searching
+1. Spectra Extraction/Quality Filtering
+1. Library Creation/Augmentation
+
+### Database Searching Proteomics Data for Library Creation
+
+Users can utilize a modified MSGF+ search workflow available at the Center for Computational Mass Spectrometry available [here](https://proteomics2.ucsd.edu/ProteoSAFe/?params=%7B%22workflow%22:%22MSGF-PLUS-AMBIGUITY%22%7D). Users may choose any public or private data they have access to and search with appropriate parameters.
+
+#### Synthetic Spectral Data Searching
+
+To search a synthetic peptide pool, please use this [workflow](https://proteomics2.ucsd.edu/ProteoSAFe/?params=%7B%22workflow%22:%22MSGF-PLUS-SYNTHETIC%22%7D). The parameters are nearly identical to MSGF+, but there are additional options. Specifically, users should add some background spectra that is not of the primary peptide pool to be identified. Users then will specify the “target_run_filename” as the full file name of the spectrum file to be identified. Identifications to other spectrum files are called decoys. Additionally, the set of peptides that that are in common between the target spectrum file and background spectrum files will be specified in “blacklisted_decoy_peptides_json” as a list of peptide strings in json format. Finally, users must enter the acceptable collision energies in the “acceptable_collision_energies” in a semicolon separated string format. All other collision energies are ignored.
+
+
+
+
+
+### Spectra Extraction/Quality Filtering
+
+After the searches are finished, the identified spectra are extracted from the original mzXML/mzML files and quality filtered. A set of search jobs can be extracted at once using this [workflow](https://proteomics2.ucsd.edu/ProteoSAFe/?params=%7B%22workflow%22:%22LIBRARY_CREATION_SPECTRA_EXTRACTION%22%7D). To select searches to extract, select an appropriate search job from the jobs list. Any jobs run by the specific user may be selected.
+
+
+![](img/massivekb/kb-extraction.png)
+
+
+### Library Creation of a New Library
+
+To create a library from an existing spectra extraction workflow, please click the “Create Library from Extracts” link. It will take you to this [workflow](https://proteomics2.ucsd.edu/ProteoSAFe/?params=%7B%22workflow%22:%22LIBRARY_CREATION_AUGMENT_LIBRARY_TEST%22%7D) with selections pre-populated. 
+
+
+![](img/massivekb/kb-create.png)
+
+The default parameters are appropriate for human HCD proteome parameters, adjust accordingly for your data.
+
+### Library Augment of an Existing Library
+
+To augment an existing library, on the status page of the existing spectral library, please click the “Augment This Library” link.
+
+![](img/massivekb/kb-augment.png)
+
+This brings users to a library augment workflow page. Please select a spectra extraction job from the jobs table to augment to the existing library. Only one extraction can be augmented to a library at one time.
```

Comment:
"augment" is not a prepositional verb, so the word "to" should be removed immediately after it. Either "augment the library" or "add to the library".